### PR TITLE
Learning Token

### PR DESCRIPTION
**Pull request by TanjinAlam**
Updated & added Learning proposal-template with following section 

- initial committers
- pre-existing repository
- contact
- web portal
- publication section

### CODE REVIEW

File: labs/learning-tokens.md
Comment (tkuhrt) [inline labs/learning-tokens.md:46]: If you would like to add a new committer to the lab, please file a ticket under [hyperledger-labs/governance issues](https://github.com/hyperledger-labs/governance/issues/new/choose) using the "Add someone to the Hyperledger Labs GitHub org" issue type.

**Review by tkuhrt — COMMENTED** [review]
There is no need to update this page, as you should be updating the lab repo instead. There is already a link to the Learning Tokens GitHub repo at the top of the page. Please let me know if you still want to update this page.

File: labs/learning-tokens.md
Comment (TanjinAlam) [reply labs/learning-tokens.md:46]: @tkuhrt Yes, I wanted to update [wiki](https://labs.hyperledger.org/labs/learning-tokens.html) page of learning token. We have recenetly made our publication and we made reference of this.

**Review by tkuhrt — APPROVED** [review]

File: labs/learning-tokens.md
Comment (AlfonsoGovela) [reply labs/learning-tokens.md:46]: Thank you @tkuhrt for your help and support. How do we get the second reviewer? Should I ask Vipin, or Kamlesh? Apologies for pushing. We are in a hurry because yesterday GBBC published our Special Report Learning Tokens, and we would like to have the readme section of our repo aligned to this document.

**Review by nidhi-singh02 — APPROVED** [review]
Lgtm!

**Comment by AlfonsoGovela** [comment]
Thank you Nidhi. We appreciate your help and support.
Kind regards,

Alfonso

**Pull request merged**
